image-builder/image_builder.sh

```diff
@@ -57,12 +57,24 @@ EOT
 exit "${error}"
 }
 
-while getopts "ho:s:" opt
+MAX_IMG_SIZE=2048
```

Comment:
Please can you either change the variable to `MAX_IMG_SIZE_MB` or add a comment explaining the units this value is in?